pill-press-app/ClientApp/src/app/models/equipment-location.model.ts

```diff
@@ -4,8 +4,8 @@ import { Application } from './application.model';
 export interface EquipmentLocation {
     id: string;
     address: CustomAddress;
-    application: Application;
+    //application: Application;
     name: string;
-    privateDwelling: string;
+    privateDwelling: boolean;
```

Comment:
This is not a boolean. Should be a string as it takes the following values: 'Yes', 'No' and 'Unknown'.

Reply:
It is actually a boolean.  The translation to Yes / No / Unknown is done in the C# code.